### ISSUE TITLE
import error

### ISSUE THREAD

**Issue by wuyadong** [avakar/pycson#2]
import error as below:

```
import cson
  File "/Users/..../lib/python3.4/site-packages/cson/__init__.py", line 1, in <module>
    ﻿from parser import load, loads
ImportError: cannot import name 'load'
```

My python version: Python 3.4.3 :: Anaconda 2.4.0 (x86_64)

I fix it manually :

__init__.py

```
from .parser import load, loads
from .peg import ParseError
```

parser.py

```
from .peg import peg
```

**Comment by avakar** [comment]
Thank you for the issue! I want Python 3 support, I merely haven't gotten to it yet. I will close the issue once I do.

**Comment by avakar** [comment]
All should be fixed in v0.3. Just update the package.
